zookeeper-server/src/main/java/org/apache/zookeeper/server/admin/Commands.java

```diff
@@ -613,7 +614,11 @@ public StatResetCommand() {
         @Override
         public CommandResponse run(ZooKeeperServer zkServer, Map<String, String> kwargs) {
             CommandResponse response = initializeResponse();
-            zkServer.serverStats().reset();
+            ServerStats serverStats = zkServer.serverStats();
```

Comment:
nit:  is there any reason, why we needed to split this into 2 lines? you are not using 'serverStats' any other place AFAIK.

Reply:
good catch. The variable was used in another place at one point during debugging, but is no longer used/needed. I reverted it to a single line.